components/dropdown/index.js

```diff
@@ -0,0 +1,40 @@
+import { PATHS } from 'constants.js';
+import React from 'react';
+import s from './s.module.css';
+import Link from 'next/link';
+
+PATHS;
+const Dropdown = () => {
+  const signout = () => {
+    fetch('https://api.realdevsquad.com/auth/signout', {
+      method: 'GET',
+      credentials: 'include',
+    }).then(() => {
+      location.reload();
+    });
+    console.log(location);
+  };
+  return (
+    <div className={`${s.dropdown}`}>
```

Comment:
This will work I guess and can you please use camelCase everywhere so you will not need the brackets to access the classNames
```suggestion
    <div className={s.dropdown}>
```

Reply:
Thanks for pointing out, I have updated the classNames to camelCase

Reply:
Please rename `s` to something meaningful

Reply:
In the latest commit I have updated the modular-styling to Styled-JSX for consistent styling. Will start naming files more meaningfully 👍🏻